Document basic usage (#85)
kyma-project/docker-registry: docs/user/tutorials/01-20-expose-registry.md

    >[!NOTE] 
    > You can find your cluster address in the `kyma-system/kyma-gateway` gateway resource.

> 1. Expose the registry service by changing the `spec.externalAccess.enabled` flag to true. Optionally, you could also change the host name:

Reviewer: ```suggestion
1. Expose the registry service by changing the **spec.externalAccess.enabled** flag to `true`. Optionally, you can also change the host name:
```

    ```bash
    kubectl apply -n kyma-system -f - <<EOF
    apiVersion: operator.kyma-project.io/v1alpha1
    kind: DockerRegistry
    metadata:
      name: default
      namespace: kyma-system
    spec:
      externalAccess:
        enabled: true
        hostPrefix: my-registry
    EOF
    ```
   

>    Once DockerRegistry CR becomes `Ready`, you will see a secret name that shall be used as `ImagePullSecret` when scheduling workloads in the cluster.

Reviewer: ```suggestion
   Once the DockerRegistry CR becomes `Ready`, you see a Secret name that is used as `ImagePullSecret` when scheduling workloads in the cluster.
```
